lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -289,7 +294,9 @@ def redcap_det_uw_reopening(*, db: DatabaseSession, cache: TTLCache, det: dict,
         specimen_received = (collection_method == CollectionMethod.SWAB_AND_SEND and \
             is_complete('post_collection_data_entry_qc', redcap_record_instance)) or \
             (collection_method == CollectionMethod.KIOSK and \
-            is_complete('kiosk_registration_4c7f', redcap_record_instance))
+            is_complete('kiosk_registration_4c7f', redcap_record_instance)) or \
```

Comment:
Let's take this opportunity to make the conditionals a little clearer with indentation. If we're not worried about 80-column widths, we can put each separate group-of-conditions (ie, collection_method == kiosk && is_complete(kiosk_registration)) on its own line, maybe?